can/common.cc

```diff
@@ -188,6 +188,22 @@ unsigned int volkswagen_crc(uint32_t address, const std::vector<uint8_t> &d) {
   return crc ^ 0xFF; // Return after standard final XOR for CRC8 8H2F/AUTOSAR
 }
 
+unsigned int volkswagen_pq_checksum(int checksum_start_bit, const std::vector<uint8_t> &d) {
+  uint8_t checksum = 0;
+  int checksum_byte = checksum_start_bit / 8;
+
+  // Simple XOR over the payload, except for the byte where the checksum lives.
+  for (int i = 0; i < d.size(); i++) {
+    if (i == checksum_byte) {
+      continue;
+    } else {
+      checksum ^= d[i];
+    }
```

Comment:
```suggestion
    if (i != checksum_byte) {
      checksum ^= d[i];
    }
```

Reply:
LOL, derp... stared at that when I wrote it for a couple minutes because I felt like there had to be a more concise way, namely, this. Don't know where my mind was. Thanks!